### DISCUSSION TITLE
DNS failure inside docker containers after jail restart

### DISCUSSION THREAD

**Discussion by undaunt** [Jip-Hop/jailmaker#183] [Q&A]
Subject, mostly. After restarting a docker enabled jail such as for adding new datasets to bind mounts, compose services that were already up and persist/relaunch upon OS restart appear to have invalid DNS.

If I bring everything down and let my dynamic networks get removed and recreated with new containers this issue is resolved.

Has anyone seen behavior like this? Thanks.

**Comment by Jip-Hop** [comment]
Some logging and examples would help. My compose stack always comes up fine without DNS issues.

**Reply by undaunt** [reply]
Agreed, though I'm not sure what to log other than curl or dig tests. When I run a dig within the jail after restart, no issues, but when I run a dig or curl against an FQDN within a container such as frigate or sonarr it fails. Is there a specific syslog or similar that may be helpful?

**Reply by Lockszmith-GH** [reply]
What to collect can be found in #135 (yes, long read, but relevant for those wanting help)

**Comment by undaunt** [comment]
I wasn't quite able to get to the bottom of this, but I moved off of local Adguard home to NextDNS so it's no longer a concern.
